### DISCUSSION TITLE
👨‍💻 Developers' Discussion 开发人员讨论

### DISCUSSION THREAD

**Discussion by lingbopro** [lingbopro/material-me#4] [Official Discussions]
# 👨‍💻 Developers' Discussion 开发人员讨论

This discussion is for developers to discuss.

For users, please go to other discussions.

这个讨论供开发者进行讨论。

对于用户，请前往其它讨论。

**Comment by lingbopro** [comment]
@Minemetero 我搓了个Tab，结果。。。

[059e086c-5853-4754-8513-14f3c71b1dab.webm](https://github.com/user-attachments/assets/9f1cd6ed-c07c-457d-83a8-72c6bbe85ff7)

**Reply by Minemetero** [reply]
感觉动画太生硬了

**Reply by lingbopro** [reply]
> 感觉动画太生硬了

毕竟是半成品

就是展示一下奇怪的特性

**Reply by lingbopro** [reply]
我终于修好了这个奇奇怪怪的bug，变量改一下位置就行了

虽然我没明白为什么（）

Commit: 87d7e197

**Reply by lingbopro** [reply]
我滴任务，完成辣！！！

[68f9d3d0-5e9f-42a1-959c-c1b1222c8320.webm](https://github.com/user-attachments/assets/f1c5c6c4-91db-4cbb-9849-612a605a9cea)

**Comment by lingbopro** [comment]
dependabot是这样的👍

![dependabot alerts](https://github.com/user-attachments/assets/9f2dfe7b-e7fd-437f-b46c-ca944039dd69)
![dependabot update pr](https://github.com/user-attachments/assets/a8b484a9-e570-4b05-89cf-f45f7481ef51)

**Reply by Minemetero** [reply]
有点意思
